src/test/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ApartmentValueFunctionTest.java

```diff
@@ -12,32 +14,51 @@
 
 class ApartmentValueFunctionTest {
 
-	@Test
-	void apartmentValueFunctionTest() throws NumberFormatException {
-
-		Apartment a = new Builder().setFloorArea(250).setAddress("108 rue de chat-ville Ville-d'Avray 92410")
-				.setNbBedrooms(1).setNbSleeping(4).setNbBathrooms(1).setTerrace(true).setFloorAreaTerrace(40)
+	ApartmentValueFunction valueFunction = new ApartmentValueFunction();
+
+	Apartment a;
+
+	/**
+	 * Inits the apartmentValueFunction object and the apartment before each tests
+	 */
+	@BeforeEach
+	void initEach() {
+		a = new Builder().setFloorArea(250)
+				.setAddress("108 rue de chat-ville Ville-d'Avray 92410")
+				.setNbBedrooms(1)
+				.setNbSleeping(4)
+				.setNbBathrooms(1)
+				.setTerrace(true)
+				.setFloorAreaTerrace(40)
 				.setDescription("Une ferme rustique en compagnie de Dwight Schrute, interdit à Jim Halpert")
-				.setTitle("Une ferme").setWifi(false).setPricePerNight(3.3).setNbMinNight(3).setTele(false).build();
-
-		ApartmentValueFunction valueFunction = new ApartmentValueFunction();
-
-		LinearValueFunction floorAreaV = new LinearValueFunction(0, 200.00);
+				.setTitle("Une ferme")
+				.setWifi(false)
+				.setPricePerNight(3.3)
+				.setNbMinNight(3)
+				.setTele(false)
+				.build();
+
```

Comment:
Utiliser before eache uniqument pour réinitialiser les valeurs entre chaque fonction

Reply:
Fixed in commit 8179cb9639e1ea27962603f4141247f023d49311